app/presenters/sufia/collection_presenter.rb

```diff
@@ -8,7 +8,7 @@ class CollectionPresenter < CurationConcerns::CollectionPresenter
     # Terms is the list of fields displayed by app/views/collections/_show_descriptions.html.erb
     def self.terms
       [:title, :total_items, :size, :resource_type, :description, :creator,
-       :contributor, :tag, :rights, :publisher, :date_created, :subject,
+       :contributor, :tags, :rights, :publisher, :date_created, :subject,
```

Comment:
Why is this being changed? Is  this is relevant to the purpose of this pull request?